### ISSUE TITLE
Unable to capture SIGINT signal when importing selenium-webdriver

### ISSUE THREAD

**Issue by atoppi** [SeleniumHQ/selenium#9059]
## 🐛 Bug Report

My nodejs script is unable to capture `SIGTERM`/`SIGINT` when being exited. By trying to find the cause of the issue I have discovered that just calling `require('webdriver-selenium')` somehow breaks the signal handling.

## To Reproduce

`index.js`
```javascript
'use strict';

const swd = require('selenium-webdriver');

function handleExit(signal) {
    console.log(`Received ${signal}. Close my server properly.`);
    process.exit(0);
}
process.on('SIGINT', handleExit);
process.on('SIGQUIT', handleExit);
process.on('SIGTERM', handleExit);

setInterval(console.log.bind(console, 'tick'), 500);
```

1. Launch `node index.js`
2. Hit `CTRL+C`

## Expected behavior

I should receive `Received SIGINT. Close my server properly.`
If I comment `const swd = require('selenium-webdriver');` everything works as expected.

## Environment

OS:  Linux (Ubuntu 20.04)
Node: v14
Language Bindings version: NodeJS 4.0.0-alpha.8

**Comment by atoppi** [comment]
**Update**: reverting to 3.6.0 solved the issue, so maybe this is a regression.

**Comment by harsha509** [comment]
Fixed with PR #9155.

Will be available in next beta release!

**Comment by ghost** [comment]
> ## 🐛 Bug Report
> My nodejs script is unable to capture `SIGTERM`/`SIGINT` when being exited. By trying to find the cause of the issue I have discovered that just calling `require('webdriver-selenium')` somehow breaks the signal handling.
> 
> ## To Reproduce
> `index.js`
> 
> ```js
> 'use strict';
> 
> const swd = require('selenium-webdriver');
> 
> function handleExit(signal) {
>     console.log(`Received ${signal}. Close my server properly.`);
>     process.exit(0);
> }
> process.on('SIGINT', handleExit);
> process.on('SIGQUIT', handleExit);
> process.on('SIGTERM', handleExit);
> 
> setInterval(console.log.bind(console, 'tick'), 500);
> ```
> 
> 1. Launch `node index.js`
> 2. Hit `CTRL+C`
> 
> ## Expected behavior
> I should receive `Received SIGINT. Close my server properly.`
> If I comment `const swd = require('selenium-webdriver');` everything works as expected.
> 
> ## Environment
> OS: Linux (Ubuntu 20.04)
> Node: v14
> Language Bindings version: NodeJS 4.0.0-alpha.8
